content/python/concepts/built-in-functions/terms/range/range.md

```diff
@@ -0,0 +1,66 @@
+---
+Title: 'range()'
+Description: 'Returns a sequence of numbers based on the given range'
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'Integers'
+  - 'Range'
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/analyze-data-with-python'
+---
+
+The `range()` function returns a sequence of numbers based on the given range.
+
+## Syntax
+
+```pseudo
+range(stop)
+range(start, stop)
+range(start, stop, step)
```

Comment:
```suggestion
range(start = 0, stop, step = 1)
```

Reply:
Just to further define the default values of some of the arguments of `range()`.